text/0000-api-version-compat.md

```diff
@@ -0,0 +1,119 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle API version support changes
+- Start Date: 2020-04-14
+- Author(s): [Emily Casey](https://github.com/ekcasey)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+
+# Summary
+[summary]: #summary
+
+In order enable smoother migrations and upgrades this RFC proposes the following changes to the API version compatibility rules:
+
+1. When the lifecycle increments an API versions in the `0.x` version line, this shall interpreted as non-breaking
+1. The spec may make certain types of "breaking" changes in a `0.x` API version, so long as those changes do not preclude simultaneous lifecycle support for the previous minor or require collaborating buildpacks to implement the same API version.
```

Comment:
2 questions:

1. Also, why is "breaking" in quotesDoes this essentially create 2 ecosystems of buildpacks?
1. Why is "breaking" in quotes?

Reply:
Words are hard. To me, breaking implies that things break, and these changes wouldn't result in breaking changes from a user perspective. I wasn't sure what word to use to describe the spec equivalent of a breaking change. I can just remove the quotes